src/controllers/hid/hidcontroller.h

```diff
@@ -65,6 +65,7 @@ class HidController final : public Controller {
     // 0x0.
     void sendBytes(const QByteArray& data) override;
     void sendBytesReport(QByteArray data, unsigned int reportID);
+	void sendFeatureReport(const QList<int>& dataList, unsigned int);
```

Comment:
```suggestion
    void sendFeatureReport(const QList<int>& dataList, unsigned int);
```
Use spaces, not tabs

Reply:
Done